.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
```

Comment:
it is redundant to have this prefix, given `kind/bug` label we add.

Reply:
Removed the title

Reply:
Please review again @pedjak